agent/src/index.test.ts

```diff
@@ -730,6 +733,26 @@ describe('Agent', () => {
         )
     }
 
+    function checkEditCodeCommand(
+        documentClient: TestClient,
+        name: string,
+        filename: string,
+        instruction: string,
+        assertion: (obtained: string) => void
+    ): void {
+        const params = { instruction: instruction }
+        checkEditCommand(documentClient, 'editCommands/code', name, filename, { params }, assertion)
```

Comment:
```suggestion
        checkEditCommand(documentClient, 'editCommands/code', name, filename, { instruction }, assertion)
```

I think this is correct, as the function takes the object like this:

```
this.registerAuthenticatedRequest('editCommands/code', params => {
            const args = { configuration: { ...params } }
            return this.createEditTask(
                vscode.commands
                    .executeCommand<FixupTask | undefined>('cody.command.edit-code', args)
                    .then(task => task && { type: 'edit', task })
            )
        })
```

We could do with some better type-safety here

Reply:
You are right, thank you very much!